backend/src/media/backends/filesystem-backend.ts

```diff
@@ -30,13 +30,17 @@ export class FilesystemBackend implements MediaBackend {
       return;
     }
     this.uploadDirectory = this.mediaConfig.backend.filesystem.uploadPath;
+    this.logger.debug(
+      `Activated media backend filesystem using ${this.uploadDirectory}`,
+    );
   }
 
   async saveFile(
     uuid: string,
     buffer: Buffer,
     fileType: FileTypeResult,
   ): Promise<string> {
+    this.logger.debug(`Using ${this.uploadDirectory}`, 'saveFile');
```

Comment:
Is this log output needed on every upload if we already have it on initialization?